packages/grafana-data/src/themes/createSpacing.ts

```diff
@@ -67,5 +75,20 @@ export function createSpacing(options: ThemeSpacingOptions = {}): ThemeSpacing {
 
   spacing.gridSize = gridSize;
 
+  // Design system spacing tokens
+  spacing.x0 = '0px';
+  spacing.x0_25 = '2px';
+  spacing.x0_5 = '4px';
```

Comment:
```suggestion
  spacing.x0.25 = '2px';
  spacing.x0.5 = '4px';
```
Would this maybe be a bit clearer?